validate.js

```diff
@@ -81,9 +81,7 @@ async function validate() {
       groupRepos[gid].push({
         name: r.name,
         fullName: fullName(r),
-        // Only include `hasRecTrack` in report.json if it's true. This is
-        // simply to match the original structure. TODO: include if false.
-        hasRecTrack: hasRecTrack ? true : undefined,
+        hasRecTrack,
```

Comment:
I'm a bit ambivalent about that one - it feels a bit noisy to have `hasRecTrack: false` everywhere, when we only really care when it's true.

Reply:
That's fine with me, I can remove the TODO without fixing it instead.

Reply:
let's go with removing the TODO for now then, thanks!

Reply:
https://github.com/w3c/validate-repos/pull/83